docs/content/administration/external-renderers.en-us.md

```diff
@@ -49,7 +49,7 @@ RUN pip3 install jupyter docutils
 
 ## `app.ini` file configuration
 
-add one `[markup.XXXXX]` section per external renderer on your custom `app.ini`:
```

Comment:
Well… According to the example in https://docs.gitea.com/administration/config-cheat-sheet#markup-markup and my personal experience, I'd say your suggested fix is wrong.

Reply:
Thanks for letting me know 
So where should I change exactly?